shoop/core/models/_product_shops.py

```diff
@@ -65,6 +65,9 @@ class ShopProduct(MoneyPropped, models.Model):
     default_price = PriceProperty('default_price_value', 'shop.currency', 'shop.prices_include_tax')
     default_price_value = MoneyValueField(verbose_name=_("Default price"), null=True, blank=True)
 
+    minimum_price = PriceProperty('minimum_price_value', 'shop.currency', 'shop.prices_include_tax')
+    minimum_price_value = MoneyValueField(verbose_name=_("Minimum price"), null=True, blank=True)
```

Comment:
All other verbose_names is in lowercase. That seems to be preferred way in other models too. Care to change that "Default price" also if not already done in master?